### ISSUE TITLE
Other Pre-trained Models

### ISSUE THREAD

**Issue by Yasintrkmn** [Jyxarthur/OCLR_model#4]
Hi,
1. Can you share your other pre-trained models?
2. What does mean 'DAVIS17-motion_test_adap_with_crf' and 'DAVIS17-motion_test_adap_without_crf' what is the difference with 'DAVIS17-motion_flow_only'

**Comment by Jyxarthur** [comment]
Thanks for your interest in our work.

1. The OCLR model is synthetically trained (therefore only one pretrained model) and can be applied to different datasets.

2. The results with "test_adap" means using the DINO mask propagation method (folder `dino/`) to refine the flow-only predictions, therefore RGB information is involved in this process. 
In detail,
"DAVIS17-motion_flow_only": Predictions from flow-only OCLR model;
"DAVIS17-motion_test_adap_without_crf": Predictions from flow-only OCLR model + DINO-based test time adaptation
"DAVIS17-motion_test_adap_with_crf": Predictions from flow-only OCLR model + DINO-based test time adaptation + an extra CRF process
